courses/free-courses-hi.md

```diff
@@ -189,6 +190,11 @@
 * [ReactJS Tutorials for Beginners In Hindi](https://www.youtube.com/playlist?list=PLUVqY59GNZQNTlOnGne0G7DXnmi7CeOtc) - Pradeep Maurya
 
 
+### Kotlin
+
+* [Kotlin Programming Complete Tutorial in Hindi](https://youtu.be/uoP4JKHgzDE) - Geeky Shows
```

Comment:
- Not a playlist but +6h video with chapters integrated in timeline
- Using title as it is
- Expanding YT url
- Adding instructor

```suggestion
* [Kotlin Programming Complete in one Video (Hindi)](https://www.youtube.com/watch?v=uoP4JKHgzDE) - Geeky Shows, `tch.:` Rajesh Kumar
```